doc/plotting.rst

```diff
@@ -743,12 +743,13 @@ This script will plot the air temperature on a map.
 
     air = xr.tutorial.open_dataset("air_temperature").air
 
-    ax = plt.axes(projection=ccrs.Orthographic(-80, 35))
-    air.isel(time=0).plot.contourf(ax=ax, transform=ccrs.PlateCarree())
-    ax.set_global()
+    p = air.isel(time=0).plot(subplot_kws=dict(projection=ccrs.Orthographic(-80, 35),
+                                               facecolor="gray"),
+                              transform=ccrs.PlateCarree())
```

Comment:
```suggestion
    p = air.isel(time=0).plot(
        subplot_kws=dict(projection=ccrs.Orthographic(-80, 35), facecolor="gray"),
        transform=ccrs.PlateCarree(),
    )
```

nit: formatting

Reply:
in the future, these kinds of formatting issues in documentation can be automatically fixed using `blackdoc`, see #4177.